src/lib/confirm-bar.js

```diff
@@ -91,23 +92,23 @@ chrome.runtime.onMessage.addListener(function({message, iconData, closeData}) {
           width: 24px;
         }
         #${confirmationId} button {
+          border: none;
           padding: 0;
-          background-color: #fbfbfb;
-          border: 1px solid #b1b1b1;
+          cursor: pointer;
+          background-color: rgba(12,12,13,0.10);
           font-family: Lucida Grande, Tahoma, sans-serif;
-          font-size: 12px;
-          height: 24px;
+          font-size: 13px;
+          height: 32px;
           width: 6em;
         }
         #${confirmationId} button:hover {
-          background-color: #ebebeb;
+          background-color: #f9f9fa;
```

Comment:
These hover & active colors for the cancel button on the confirm bar don't seem right, but the existing ones just kind of merged into the background

Reply:
@sevaan ?